atmosphere/settings/local.py.j2

```diff
@@ -575,9 +575,17 @@ WEB_DESKTOP = {
     }
 }
 
+{% if TESTING %}
+CACHES = {
+    'default': {
+        'BACKEND': 'django.core.cache.backends.dummy.DummyCache',
+    }
+}
```

Comment:
This could lead to unintended consequences.. `TESTING` previously _enabled_ the ability to run `./manage.py test`. 

Rather than overloading `TESTING` perhaps we could go [this route](https://docs.djangoproject.com/en/1.11/topics/testing/tools/#overriding-settings) instead?

Reply:
Wouldn't be the first time I'm confused about the historical purpose for a variable/setting. The problem is we probably need caching disabled for _all_ tests - so it becomes very tedious very quickly. But maybe your idea of making the caching backend configurable is the answer.